src/nvapi_globals.h

```diff
@@ -4,8 +4,10 @@
 #include "nvapi/resource_factory.h"
 #include "nvapi/nvapi_adapter_registry.h"
 #include "nvapi/nvapi_d3d_instance.h"
+#include "nvapi/vk.h"
 
 extern uint64_t initializationCount;
 extern std::unique_ptr<dxvk::ResourceFactory> resourceFactory;
 extern std::unique_ptr<dxvk::NvapiAdapterRegistry> nvapiAdapterRegistry;
 extern std::unique_ptr<dxvk::NvapiD3dInstance> nvapiD3dInstance;
+extern std::unique_ptr<dxvk::Vk> vkLowLatency;
```

Comment:
Since this `Vk` instance is only needed for `NvapiVulkanLowLatencyDevice`, we might also keep it alive next to `static std::unordered_map<VkDevice, NvapiVulkanLowLatencyDevice> m_lowLatencyDeviceMap;` as another static field of `NvapiVulkanLowLatencyDevice` and initialize as part of e.g. `GetOrCreate(ResourceFactory& resourceFactory, VkDevice device)`.

Reply:
I gave this a thought and I'm not sure about this. I wanted `NvapiVulkanLowLatencyDevice` to not be aware of any other implemetation detail like `ResourceFactory` so I kept `NvAPI_Vulkan_InitLowLatencyDevice` as the only piece that connects both worlds; accepting just `vkGetDeviceProcAddr` made it standalone and very self-contained.

But if you really want it like this, I'd suggest separating `GetOrCreate(ResourceFactory& resourceFactory, VkDevice device)` into `InitializeVulkan(ResourceFactory& resourceFactory)` and `GetOrCreate(VkDevice device)`.

Reply:
ok, I get your point wrt keeping `NvapiVulkanLowLatencyDevice` clean. My reasoning was that having `std::unique_ptr<dxvk::Vk> vkLowLatency` in `globals` looks somewhat "naked", especially since the other part of the static application skeleton for vkreflex, the list of created `NvapiVulkanLowLatencyDevice`'s, lives in `NvapiVulkanLowLatencyDevice` itself. Also looking at other parts, on the sysinfo side we have the registry for connecting adapters and entrypoints, which also hold the `Vk` instance , for lfx there is `NvapiD3dInstance` which hold the `Lfx` instance. 
Creating  `Initialize(ResourceFactory& resourceFactory)` and `GetOrCreate(VkDevice device)` sounds also fine of course and it fits even better.
If you dislike this approach though, we can keep your initial version. But if I had to choose, I would go for `Initialize()` and `GetOrCreate()` in `NvapiVulkanLowLatencyDevice`.

Reply:
I think I'll go with `Initialize` then.